index.js

```diff
@@ -3,5 +3,9 @@ const search = require('./lib/search')
 const projects = require('./data/projects.json')
 
 module.exports = async function (project) {
-  return search(projects[project].q)
+  if (project in projects) {
+    return search(projects[project].q)
+  } else {
+    return search(`${project.includes('/') ? 'repo' : 'org'}:${project} state:open label:"good first issue"`)
```

Comment:
By the by - this check for the `/` character does actually work, because GitHub won't allow repository names with a `/`:

![image](https://user-images.githubusercontent.com/10660468/50049941-713aab00-00bd-11e9-9993-341b9785ecc5.png)